### ISSUE TITLE
gopass otp progressbar formatting bug when invoked with 0 seconds left

### ISSUE THREAD

**Issue by defhacks** [gopasspw/gopass#2046]
### Summary

gopass otp prints mangled progress bar when token has 0 seconds left at time of invocation.
e.g.

```
⚠ This OTP password still lasts for:
] %!(NOVERB)%!(EXTRA int64=0, int64=0)[Goooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooopass] 100.00%
```

### Steps To Reproduce

run gopass otp <secret> when time left for token is 0. I just looped using Ctrl+C and re-running manually.

### Expected behavior

non-corrupt output in the progress bar

### Environment

- OS:  Ubuntu 20.04
- gopass Version: gopass 1.13.0+c574e920b532606894b3e64db059f3e3a838b0c9
- Installation method: gopass update, iirc ?

### Additional context

I tracked it down to the nested `Sprintf()` calls here: https://github.com/gopasspw/gopass/blob/master/pkg/termio/progress.go#L133

I was curious why this was happening and ended up being able to figure out what's going by building an isolated demo of the problematic code path, but I'm not clear what the best fix is, likely some special casing of `max = 0` ?

Should be `go run`-able, just add imports
```
func main() {
        max := 1
        digits := int(math.Log10(float64(max))) + 1
        fmt.Println(digits)
        fmt_string := fmt.Sprintf(" %%%dd / %%%dd ", digits, digits)
        fmt.Println(fmt_string)

        max = 0
        digits = int(math.Log10(float64(max))) + 1
        fmt.Println(digits)
        fmt_string = fmt.Sprintf(" %%%dd / %%%dd ", digits, digits)
        fmt.Println(fmt_string)
}
```
output:
```
1
 %1d / %1d
-9223372036854775807
 %-9223372036854775807d / %-9223372036854775807d
```

**Comment by AnomalRoil** [comment]
Shoot, I guess we probably have a edge-case there. 

Do you want to add the special handling for the case where it's 0 yourself in a PR or shall I?

**Comment by AnomalRoil** [comment]
Fixed in #2048
